lib/bundle/brew_installer.rb

```diff
@@ -132,6 +135,14 @@ def link_change_state!(verbose: false)
       true
     end
 
+    def postinstall_change_state!(verbose:)
+      return true if @postinstall.blank?
+      return true unless changed?
+
+      puts "Running postinstall for #{@name}." if verbose
+      Bundle.system(@postinstall, verbose:)
```

Comment:
Does this work with the README example where it's all one string rather than separate arguments? `Bundle.system` intentionally avoids shell execution so may need another method.

Reply:
@Bo98 Yup, still works, that's what I tested locally.